src/update.c

```diff
@@ -79,7 +82,23 @@ static struct list *full_download_loop(struct list *updates, int isfailed)
 		if (file->is_deleted) {
 			continue;
 		}
-
+		/* Mix content is local, so don't queue files up for curl downloads */
+		if (file->is_mix) {
+			string_or_die(&url, "%s/%i/files/%s.tar", MIX_STATE_DIR, file->last_change, file->hash);
```

Comment:
This is not a URL. Needs a better variable name.

Reply:
Similarly for this comment, I'm going to ignore my concern about variable-name appropriateness for now.